core/src/main/java/org/apache/accumulo/core/client/admin/NamespaceOperations.java

```diff
@@ -230,6 +234,9 @@ default Iterable<Entry<String,String>> getProperties(String namespace)
   Map<String,String> getConfiguration(String namespace)
       throws AccumuloException, AccumuloSecurityException, NamespaceNotFoundException;
 
+  Map<String,String> getNamespaceSpecificConfiguration(String namespace)
```

Comment:
Is this meant to return the configuration stack (default -> system -> namespace) or only the properties for the namespace?  I think we should differentiate when dealing with properties (specific prop name, values for one id) vs configuration when it is the configuration hierarchy.

Reply:
This is only meant to return the configuration that can be mutated in Zookeeper for the namespace so I'm going to rename this to just getNamespaceConfiguration() as Dave suggested yesterday. The getConfiguratoin() method that's above already returns everything.